### DISCUSSION TITLE
Can't import the named export 'buildLocalizeFn' from non EcmaScript module (only default export is available)

### DISCUSSION THREAD

**Discussion by jayp-dev** [gpbl/react-day-picker#2053] [Support]
I am Gotting this error Can you someone Help me 
 
Failed to compile.

./node_modules/date-fns/locale/ug/_lib/localize.mjs
Can't import the named export 'buildLocalizeFn' from non EcmaScript module (only default export is available)
![Screenshot from 2024-02-07 10-39-11](https://github.com/gpbl/react-day-picker/assets/70154315/4f407270-3bc7-40ed-aa15-4725f7f5ea4d)

Thanks

**Comment by gpbl** [comment]
Hi @jayp-dev, have you checked the answers to this question on Stack Overflow? https://stackoverflow.com/questions/69343038
